warehouse/postgres/postgres.go

```diff
@@ -191,6 +191,12 @@ func (pg *HandleT) DownloadLoadFiles(tableName string) ([]string, error) {
 }
 
 func (pg *HandleT) loadTable(tableName string, tableSchemaInUpload warehouseutils.TableSchemaT, skipTempTableDelete bool) (stagingTableName string, err error) {
+	sqlStatement := fmt.Sprintf(`SET search_path to "%s"`, pg.Namespace)
```

Comment:
should we use search_path or just mention schema name in create table query ? you mentioned somthing about it not using search path in few cases even though it was set, right?